airbyte-webapp/src/components/StateBlock/StateBlock.tsx

```diff
@@ -0,0 +1,169 @@
+import { highlight, languages } from "prismjs/components/prism-core";
+import React, { useState, useEffect, useCallback } from "react";
+import { FormattedMessage } from "react-intl";
+import "prismjs/components/prism-clike";
+import "prismjs/components/prism-javascript";
+import "prismjs/themes/prism.css";
+import Editor from "react-simple-code-editor";
+import styled from "styled-components";
+
+import { H5, LoadingButton } from "components";
+import ContentCard from "components/ContentCard";
+
+import { ConnectionState, ConnectionStateObject } from "core/request/AirbyteClient";
+import { useConfirmationModalService } from "hooks/services/ConfirmationModal";
+import { useGetConnectionState, useUpdateConnectionState } from "hooks/services/useConnectionHook";
+
+type IProps = {
+  connectionId: string;
+};
+
+const StateBlockComponent = styled(ContentCard)`
```

Comment:
We're no longer using styled-component for new code, this should be moved to a SCSS module instead.